airflow/providers/google/cloud/hooks/cloud_storage_transfer_service.py

```diff
@@ -98,6 +105,22 @@ class GcpTransferOperationStatus:
 NEGATIVE_STATUSES = {GcpTransferOperationStatus.FAILED, GcpTransferOperationStatus.ABORTED}
 
 
+def gen_job_name(job_name: str) -> str:
+    """
+    Adds unique suffix to job name. If suffix already exists, updates it.
+    Suffix — current timestamp
+    :param job_name:
+    :rtype job_name: str
+    :return:
+    """
+    split = job_name.split("_")
+    uniq = str(int(time.time()))
+    if len(split) > 1 and re.compile("^[0-9]{10}$").match(split[-1]):
+        split[-1] = uniq
+        return "_".join(split)
+    return "_".join([job_name, uniq])
```

Comment:
Can't we just return `return f"{job_name}_{uniq}`?

Reply:
Sure we can, i've coded it before your advise. I'll fix it